### DISCUSSION TITLE
Webhooks

### DISCUSSION THREAD

**Discussion by Nic-J** [crowdin/crowdin-api-client-go#38] [Ideas]
Would be great to add methods to manage webhooks (project & organization webhooks)

**Comment by andrii-bodnar** [comment]
@Nic-J Thanks for the suggestion! We plan to implement these APIs soon as well.

**Reply by andrii-bodnar** [reply]
@Nic-J available in [v0.4.0](https://github.com/crowdin/crowdin-api-client-go/releases/tag/v0.4.0)
